### ISSUE TITLE
Functionality of the screen is operable through Tab and arrow kays but there is no such indication, Publish issue.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#805]
**Describe the bug**
Functionality of the screen is operable through Tab and arrow kays but there is no such indication, Publish issue.

**To Reproduce**

1. Go to Publish  (http://localhost:9992/cm/app/?view=publish&site=test2&mode=readonly&name=index.html&path=%2FSites%2Ftest2%2F&pathType=page)
2. Create on Add New Server or Server Sitting. It will open a pop up.
3. On this pop up Folder Location and Server Default radio buttons are accessible through Tab and Arrow key but not indicated.

**Expected behavior**
if Functionality of the screen is operable through Tab and arrow kays then indication should be there.

**Additional context**
Navigate all the options available for similar issue and fix.
